DSCResources/cChocoPackageInstall/cChocoPackageInstall.psm1

```diff
@@ -292,16 +295,26 @@ function IsPackageInstalled
 }
 
 Function Test-LatestVersionInstalled {
+    [Diagnostics.CodeAnalysis.SuppressMessage('PSAvoidUsingInvokeExpression','')]

```

Comment:
I added the use of Invoke-Expression to `Test-LatestVersionInstalled`, since the other internal functions (`InstallPackage` / `Upgrade-Package`),  seem to use Invoke-Expression as well and that specific `PSScriptAnalyzer` rule will cause the build to fail. 
I don't mind changing all uses of Invoke-Expression to `Start-Process` in this pull request, but this might be better to do in a different commit, just let me know.